pkg/urbit/noun/manage.c

```diff
@@ -113,17 +113,15 @@ _cm_punt(u3_noun tax)
 static void
 _cm_emergency(c3_c* cap_c, c3_l sig_l)
 {
-  c3_i ret_i;
-
-  ret_i = write(2, "\r\n", 2);
-  ret_i = write(2, cap_c, strlen(cap_c));
+  c3_write(2, "\r\n", 2);
+  c3_write(2, cap_c, strlen(cap_c));
 
   if ( sig_l ) {
-    ret_i = write(2, ": ", 2);
-    ret_i = write(2, &sig_l, 4);
+    c3_write(2, ": ", 2);
+    c3_write(2, &sig_l, 4);
   }
 
-  ret_i = write(2, "\r\n", 2);
+  c3_write(2, "\r\n", 2);
```

Comment:
This function is an "emergency printf", called from within arbitrary (ie, async) signal handlers. I don't think we need or want retry here.